Felienne/hedy
# [BUG FIX] Username on explore page box is truncated correctly

**Description**
In this PR we lower the truncate threshold on the username within the explore program box. This way we make sure that the username isn't "out-of-line" and is completely shown on a new line.

**Fixes**
This PR fixes #2661.

**How to test**
Create an account with a username > 25 characters. Verify that the username is correctly truncated on the explore page when having a shared program.
